### PR TITLE
Cast and normalize template input strings

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

File: backend/fms_core/template_importer/importers/container_creation.py [hunk 1]
```diff
@@ -14,12 +16,12 @@ def import_template_inner(self):
         for row_id, row_data in enumerate(containers_sheet.rows):
             container = {
                 'kind': row_data['Container Kind'],
```
Comment (sebastianballesteros) [inline backend/fms_core/template_importer/importers/container_creation.py:18]: Do we wanna `str_cast_and_normalize` Container Kind?

Comment (UlysseFG) [reply backend/fms_core/template_importer/importers/container_creation.py:18]: If this is a string we should i think. This is unlikely someone will input a number in there but better safe than sorry.

Comment (ckostiw) [reply backend/fms_core/template_importer/importers/container_creation.py:18]:
```suggestion
'kind': str_cast_and_normalize(row_data['Container Kind']),
```

Comment (ckostiw) [reply backend/fms_core/template_importer/importers/container_creation.py]:
```suggestion
                'kind': str_cast_and_normalize (row_data['Container Kind']),
```

Comment (UlysseFG) [reply backend/fms_core/template_importer/importers/container_creation.py:18]: Oh right ... i should apply that !

File: backend/fms_core/template_importer/importers/sample_submission.py [hunk 2]
```diff
@@ -19,49 +20,49 @@ def import_template_inner(self):
...
-                'alias': row_data['Individual Alias'],
-                'sex': row_data['Sex'],
-                'pedigree': row_data['Pedigree'],
-                'taxon': row_data['NCBI Taxon ID #'],
-                'cohort': row_data['Cohort'],
+                'name': str_cast_and_normalize(row_data['Individual ID']),
+                'alias': str_cast_and_normalize(row_data['Individual Alias']),
+                'sex': str_cast_and_normalize(row_data['Sex']),
+                'pedigree': str_cast_and_normalize(row_data['Pedigree']),
+                'taxon': int(row_data['NCBI Taxon ID #']),
```
Comment (ckostiw) [inline backend/fms_core/template_importer/importers/sample_submission.py:36]: int(None) and int('') will throw an exception. Should the int conversion be put in a function that handles the exception and returns None?

**Review by ckostiw — CHANGES_REQUESTED** [review]

File: backend/fms_core/template_importer/importers/sample_submission.py (hunk 2 above)
Comment (UlysseFG) [reply backend/fms_core/template_importer/importers/sample_submission.py:36]: yeah i'll add something to the utils for that...

**Review by ckostiw — APPROVED** [review]

**Pull request merged**